e2e/types/chain.go

```diff
@@ -24,7 +25,7 @@ var (
 		IsPublic:           true,
 		BlockPeriod:        15 * time.Second,
 		FinalizationStrat:  netconf.StratLatest,
-		AVSContractAddress: contracts.TestnetAVS(),
+		AVSContractAddress: common.HexToAddress("0xa7b2e7830C51728832D33421670DbBE30299fD92"),
```

Comment:
lets remove this from `types.EVMChain` and from `netconf.Chain` and move it to `netconf.ID.Static()`.

So add `AVSContractAddress` as a field to `netconf.Static`.

Reply:
I added just the `AVSContractAddress ` field in the testnet. I don't want to refactor the rest of the contracts now with the input EVMChain - this is the bare minimum that the Monitor service needs.